templates/build.sh.j2

```diff
@@ -23,6 +23,9 @@ function buildBinaries() {
     make update OVERRIDE=1
     make nimbus_beacon_node \
         LOG_LEVEL="{{ beacon_node_build_log_level }}" \
+        {% if beacon_node_nim_commit != "" %}
+        NIM_COMMIT={{ beacon_node_nim_commit }} \
```

Comment:
"NIM_COMMIT=..." should be duplicated in `make update`, above, because that's where the compiler is built.

No point building the default compiler version, if we are not going to use it.